src/storages/inRedis/uniqueKeysCacheInRedis.ts

```diff
@@ -0,0 +1,50 @@
+import { IUniqueKeysCacheBase } from '../types';
+import { Redis } from 'ioredis';
+import { UniqueKeysCacheInMemory } from '../inMemory/uniqueKeysCacheInMemory';
+import { setToArray } from '../../utils/lang/sets';
+
+const DEFAULT_CACHE_SIZE = 30000;
```

Comment:
there are multiple occurrences of `const DEFAULT_CACHE_SIZE = 30000;` and 2 occurrences of `_REFRESH_RATE = 300000; // 300.000 ms = start after 5 mins`
We can move it to `/inRedis/constants.ts`